crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -121,16 +122,16 @@ pub struct LiquidityRequest {
 }
 
 /// An LN-DLC node.
-pub struct Node<S> {
+pub struct Node<S: TenTenOneStorage, N: Storage> {
```

Comment:
I only added it because clippy required it.

I think it stems from the fact that I switch from dyn trait arguments to a generic, which then had to be specified on all of our types. I don't think the constraints changed at all, it was just a different syntax.

Reply:
But the change leaks through the entire crate as we now have to define the trait bounds everywhere.

I do think there isn't much we can do about it. Favouring static dispatch should make sense in terms of performance, although benchmarking could be interesting.